source/dpaa_shutdown_bepa.rst

```diff
@@ -98,7 +98,7 @@ Our real-time web pages will alert us and the Lead System Engineer will call us.
 
 .. note::
 
-   As of this writing, the ACIS Flight Software Patch level is standard G, optional H, version = 56. 
+   As of this writing, the ACIS Flight Software Patch level is standard G, optional I, version = 58. 
    Before preparing the CAP, check that this is the correct version.
 
 Impacts
```

Comment:
The statement on the TXINGS parameters needs to be stronger.  We no longer want to wait for a SAR but would like to update parameters by CAP.